code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -16,7 +16,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/pistol/m1911
-	name = "\improper M1911 pistol"
+	name = "M1911"

```

Comment:
yeah, it's inconsistent with the other naming changes in the pull

not everyone knows that M1911 is a 9mm pistol